src/XNGMarkdownParser.m

```diff
@@ -319,28 +337,16 @@ - (void)consumeToken:(XNGMarkdownParserCode)token text:(char *)text {
             }
             break;
         }
-        case MARKDOWN_BULLETSTART: {
-            NSInteger numberOfDashes = [textAsString rangeOfString:@" "].location;
-            if (_bulletStarts.count > 0 && _bulletStarts.count <= numberOfDashes) {
-                // Treat nested bullet points as flat ones...
-
-                // Finish off the previous dash and start a new one.
-                NSInteger lastBulletStart = [[_bulletStarts lastObject] intValue];
-                [_bulletStarts removeLastObject];
-
-                [_accum addAttributes:[self paragraphStyle]
-                                range:NSMakeRange(lastBulletStart, _accum.length - lastBulletStart)];
-            }
-
-            [_bulletStarts addObject:@(_accum.length)];
-            textAsString = @"•\t";
-            break;
-        }
         case MARKDOWN_NEWLINE: {
             textAsString = @"";
             break;
         }
         case MARKDOWN_URL: {
+            
+            if ([textAsString hasPrefix:@"@"]) {
```

Comment:
this for another issue (#17) reported and I also encountered, domain in email address shouldn't not be rendered as link. I tried to do pure regex fix, no good. ended up with let regex expend optional leading at sign (@) , and let code to skip domain in email.